pkgs/tools/misc/flashrom/default.nix

```diff
@@ -20,20 +23,36 @@ stdenv.mkDerivation rec {
     hash = "sha256-iaf/W+sIyJuHlbvSU6UblFNUeoZMMXkzAilrVrvFbWU=";
   };
 
-  nativeBuildInputs = [ pkg-config installShellFiles ];
-  buildInputs = [ libftdi1 libusb1 pciutils ]
-    ++ lib.optional jlinkSupport libjaylink;
+  nativeBuildInputs = [
+    meson
+    ninja
+    pkg-config
+    installShellFiles
+  ];
 
-  postPatch = ''
-    substituteInPlace util/z60_flashrom.rules \
-      --replace "plugdev" "flashrom"
-  '';
+  buildInputs = [
+    cmocka
+    libftdi1
+    libusb1
+    pciutils
+  ] ++ lib.optional jlinkSupport libjaylink;
 
-  makeFlags = [ "PREFIX=$(out)" "libinstall" ]
-    ++ lib.optional jlinkSupport "CONFIG_JLINK_SPI=yes";
+  # These options were reworked in the 1.3.0 release,
+  # we'd instead say something like:
+  #   let programmers = lib.concatStringsSep "," ([ "auto" ] ++ optional jlinkSupport [ "_jlink_" ];
+  #   in "-Dprogrammer=${programmers};"
+  mesonFlags = [
+    (lib.strings.mesonBool "-Dconfig_jlink_spi" jlinkSupport)
```

Comment:
I did not see `config_jlink_spi` in https://github.com/flashrom/flashrom/blob/v1.2.1/meson_options.txt

Reply:
Ah damn, it only got introduced in 1.3.0, not sure how I missed this.

Right, so we'll have to wait for a new release I guess, I'll close this PR, sorry for the noise.

Reply:
The Meson build system is not stable with 1.3.0 either.